test/test_jit.py

```diff
@@ -1023,7 +1023,7 @@ def forward(self, x):
         m = torch.jit.script(M())
         observer = torch.jit.script(default_observer())
         qconfig_dict = {'': script_qconfig(default_qconfig)}
-        torch._C._jit_pass_insert_observers(m._c, "forward", qconfig_dict, True)
+        m = wrap_cpp_module(torch._C._jit_pass_insert_observers(m._c, "forward", qconfig_dict, False))
```

Comment:
Can you provide more context on why this change is needed? Only the test has been changed without any change in the code, so want to understand why.

Reply:
this is make sure tests don't modify the original ClassType inplace so that it does not affect other tests in the system